.github/workflows/diff-javadoc.yml

```diff
@@ -0,0 +1,58 @@
+name: Diff Javadoc
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Make Dir
+        run: mkdir ~/diff
+
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+
+      - name: Set up JDK 11
```

Comment:
i think setup would be better done first.

Reply:
Setup needs gradle files to be present, so they need to be checked out